packages/core/src/odata-v2/legacy-request-configs/odata-function-import-request-config.ts

```diff
@@ -1,39 +0,0 @@
-import { oDataUriV2 } from '../uri-conversion';
-import {
-  RequestMethodType,
-  FunctionImportParameters,
-  ODataFunctionImportRequestConfig as Base
-} from '../../odata-common';
-
-/**
- * @deprecated Since v1.21.0. Use superclass instead.
- * OData function import request configuration for a set of parameters.
- *
- * @typeparam ParametersT - Type of the original parameters object
- */
-class ODataFunctionImportRequestConfig<ParametersT> extends Base<ParametersT> {
```

Comment:
This class is exported with new name, see the last line of this file.
If I remember correctly, we keep them for compatibility reasons.
@marikaner , please correct me.

Reply:
I restored the file and changed it to extend the V2 version of the config.